src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table-actions/table-actions.component.html

```diff
@@ -12,8 +12,8 @@
   <button type="button"
           dropdownToggle
           *ngIf="showDropDownActions()"
-          class="btn btn-sm btn-primary dropdown-toggle dropdown-toggle-split">
-    <ng-container *ngIf="onlyDropDown">{{ onlyDropDown }}</ng-container>
+          class="btn btn-sm btn-{{btnColor}} dropdown-toggle dropdown-toggle-split">
```

Comment:
Despite I don't think this cleanup/optimization belongs into this single commit, I don't think the optimization is necessary at all. But if you have strong feellings about it, could you please do it for the whole project, instead of just the files you've had to adapt in order to complete the task? I think a separate PR makes most sense here.

Reply:
My mistake! I didn't understood it correctly, thanks for the explanation!  :grin: